spec/ndx-optogenetics.extensions.yaml

```diff
@@ -329,3 +329,6 @@ groups:
     dtype: float
     doc: Constant power of excitation source throughout the epoch, in mW, e.g., 77
       mW.
+  - name: optical_fiber_locations_table_region
```

Comment:
What do you think about renaming this `optical_fiber_location`? We usually name the columns as a singular noun and leave out the `table_region` part, but it is kind of nice that it is explicitly signaled that this is a DynamicTableRegion. The name is just getting a bit long and unwieldy.

Reply:
Usually in the extension

Reply:
Okay, I added `optical_fiber_locations_index` as a `VectorIndex` column, but I'm not sure how to add it row-wise...

With the current code in test_example_usage, I get the following warning: 
```python
src/pynwb/tests/test_example_usage.py::test_example_usage
  /opt/anaconda3/envs/dan_lab_to_nwb_env/lib/python3.13/site-packages/hdmf/build/objectmapper.py:893: IncorrectDatasetShapeBuildWarning: Shape of data does not match shape in spec 'DynamicTableRegion'
    warnings.warn(msg, IncorrectDatasetShapeBuildWarning)
```

Do you know what I'm doing wrong here?

Reply:
> You don't need to add it row-wise. It's contents will be managed by DynamicTable. So you can remove:

If I omit it, I get the following warning:
```python
src/pynwb/tests/test_example_usage.py::test_example_usage
  /opt/anaconda3/envs/dan_lab_to_nwb_env/lib/python3.13/site-packages/hdmf/build/objectmapper.py:706: MissingRequiredBuildWarning: OptogeneticEpochsTable 'optogenetic_epochs' is missing required value for attribute 'optical_fiber_locations_index'.
    warnings.warn(msg, MissingRequiredBuildWarning)
```

But with the current code, it works without warning/error.

Reply:
I will investigate

Reply:
Ah, I see now. Thanks for the fix!